features/openhab-addons-legacy/src/main/feature/feature.xml

```diff
@@ -122,6 +122,12 @@
     <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.rwesmarthome/${project.version}</bundle>
   </feature>
 
+  <feature name="openhab-binding-samungtv1" description="Samsung TV Binding (1.x)" version="${project.version}">
```

Comment:
When you are dealing with it tonight, you might also want to replace "samungtv1" with "samsungtv1", so that [the build won't break](https://openhab.ci.cloudbees.com/job/PR-openHAB-Distribution/522/console) any longer ;-)

Reply:
I hope this was not the reason my 1.x tests did fail, maybe I will look into it later but for now it becomes to me unlikely that the binding was part of the distribution recently and I get back to thinking I broke something :-)